sample_files/problem_setting/test/run_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+BLUE="\e[0;34m"
+GREEN="\e[0;32m"
+WHITE="\e[0m"
+
+for suite in $(ls testsuite); do
+    if ! g++ testsuite/$suite/main.cpp -I.. -o test; then
+        echo "Compilation of $suite failed."
+        return 1
+    fi
+    echo -e ${BLUE}"===Suite $suite==="${WHITE}
+    for testcase in $(ls testsuite/$suite/cases); do
```

Comment:
There are so many things wrong with this...

Reply:
Fixed